### ISSUE TITLE
Parameters Broke on GDScript

### ISSUE THREAD

**Issue by AlexDarigan** [AlexDarigan/WAT#286]
I think they still work, they're just being overwritten.

**Comment by 3ter** [comment]
Just a short clarification: Do you mean parameterized tests as described in the [example](https://github.com/AlexDarigan/WAT/blob/main/tests/examples/gdscript/example.test.gd) currently don't work as intended?

**Comment by AlexDarigan** [comment]
When using the GUI, the GDScript version of parameterized tests only display one run of the tests instead of all runs. This is a graphical error only (and I believe should one of the runs fail, the method will still be shown as failing).

**Comment by 3ter** [comment]
I'm getting only one test as well when using the CLI (in addition to some errors), I've used the example linked to above.
```
$ ~/Downloads/Godot_v3.3.3-stable_x11.64 addons/WAT/cli.tscn run=all
Godot Engine v3.3.3.stable.official.b973f997f - https://godotengine.org
OpenGL ES 3.0 Renderer: GeForce GTX 1070/PCIe/SSE2
OpenGL ES Batching: ON
 

-------RESULTS-------
Took 0 second(s)
0 Tests Crashed
1 / 1 Tests Passed
-------RESULTS-------

ERROR: ~List: Condition "_first != __null" is true.
   At: ./core/self_list.h:112.
ERROR: ~List: Condition "_first != __null" is true.
   At: ./core/self_list.h:112.
WARNING: cleanup: ObjectDB instances leaked at exit (run with --verbose for details).
   At: core/object.cpp:2132.
ERROR: clear: Resources still in use at exit (run with --verbose for details).
   At: core/resource.cpp:450.
```

**Comment by AlexDarigan** [comment]
Tests in the CLI mean Scripts. Test Methods would only appear as failed methods.

However it does seem there is a logic side to this so they may be proper broken.

**Comment by AlexDarigan** [comment]
@3ter Should be resolved in 4f4ffc5bc1098829ba9eb62a5ff273213e02ee23.

I was using a for loop for test execution which kept ending things too early. Add a few rerun checks in there too, should be all good now.
